app-frontend/src/app/pages/admin/platform/settings/email/email.html

```diff
@@ -39,26 +40,42 @@ <h1 class="h3 page-title">Email Notifications</h1>
       <span class="error color-danger" ng-show="emailSettings.password.$error.required">required</span>
     </div>
     <div class="form-group">
-      <label for="emailfrom">From</label>
+      <label for="emailfrom">From Field</label>
       <input id="emailfrom"
-             type="text"
+             type="email"
```

Comment:
I'm pretty sure this was done on purpose - the from field should be able to say something like "Raster Foundry Support", which isn't an email

Reply:
The from field has to be in the email format since this is used in `setFrom(param1, param2)`'s `param1`, which requires a domain, and which caused the issue mentioned in https://github.com/raster-foundry/raster-foundry/issues/4323. I added a required display name field in the latest commits to be passed to `param2`, which can be something like `Raster Foundry Support`.